apps/api/src/controllers/v1/scrape.ts

```
@@ -0,0 +1,149 @@
+    if (
+      e instanceof Error &&
+      (e.message.startsWith("Job wait") || e.message === "timeout")
+    ) {
+      return res.status(408).json({
+        success: false,
+        error: "Request timed out",
+      });
+    } else {
+      return res.status(500).json({
+        success: false,
+        error: `(Internal server error) - ${e && e.message ? e.message : e}`,
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)